packages/jest-cli/src/reporters/CoverageReporter.js

```diff
@@ -103,11 +103,13 @@ class CoverageReporter extends BaseReporter {
       aggregatedResults.coverageMap = map;
     } catch (e) {
       console.error(
-        chalk.red(`
+        chalk.red(
+          `
```

Comment:
The original code was likely ``chalk.red(` `` and the first time the project was converted to prettier, it got (incorrectly) moved to the next line. But now that prettier is smart about it, we should put it where it belonged in the first place and prettier will respect that.